requirements.txt

```
@@ -6,5 +6,10 @@ django==3.0.6
 gunicorn==20.0.4
 psycopg2-binary==2.8.5
 python-memcached==1.59
-social-auth-app-django==3.1.0
 whitenoise==5.0.1
+django-allauth==0.41.0
+bootstrap_4==0.1.0
```

this isn't a pypi package. This throws an error, you want `django-bootstrap4`
```suggestion
django-bootstrap4
```

---

Thanks, corrected.